controller/readwrite_test.go

```diff
@@ -26,7 +26,7 @@ func TestReadWriteRun(t *testing.T) {
 		applyTaskErr      error
 		resolverRunErr    error
 		templateRenderErr error
-		watcherWaitErr    error
+		taskName          string
```

Comment:
Note: I made 2 additional updates to this test
 - removed the case field for watcher.Wait: this method no longer calls watcher.Wait
 - added the case for template.Render